### PR TITLE
chore: bump @microsoft/api-extractor to get the latest version of path-parse

### PR DESCRIPTION
**Pull request by OrkoHunter**
https://snyk.io/vuln/SNYK-JS-PATHPARSE-1077067

No changeset needed since this is part of the mono-repo setup.

### CODE REVIEW

**Comment by benjdlambert** [comment]
Looks like we are bound to this: https://github.com/microsoft/rushstack/pull/1916 and using the npm package from this build

**Comment by benjdlambert** [comment]
Looks like there's been some action in the last few days though - maybe a release is on the cards.

**Comment by freben** [comment]
yeah all of these three have new releases but not quite all the way there yet for us - this is what i poked at this morning and mentioned; it leads to errors where `import()` statements in `.d.ts` files choke the processing

File: plugins/circleci/api-report.md [hunk 1]
```diff
@@ -49,33 +51,28 @@ export const circleCIApiRef: ApiRef<CircleCIApi>;
 export const circleCIBuildRouteRef: RouteRef<undefined>;
 
 // @public (undocumented)
-const circleCIPlugin: BackstagePlugin<{}, {}>;
-
+const circleCIPlugin: BackstagePlugin<    {}, {}>;
```
Comment (benjdlambert) [inline plugins/circleci/api-report.md:54]: There's some weird stuff going on with them here...

Comment (OrkoHunter) [reply plugins/circleci/api-report.md:54]: Haha

Comment (benjdlambert) [reply plugins/circleci/api-report.md:54]: Maybe we just run prettier on these `api-reports.md` after the build. They're currently excluded for some reason right now, guessing maybe that we didn't want the warnings on them at build time.

**Comment by OrkoHunter** [comment]
Closed in favor of https://github.com/backstage/backstage/pull/6433

**Pull request closed without merging**